### DISCUSSION TITLE
How to change the VIP in deployed cluster?

### DISCUSSION THREAD

**Discussion by vasilytray** [autobase-tech/autobase#330] [Q&A] [answered]
please tell me, if after installing the cluster I need to change the VIP, how do I do it on an already deployed cluster?

**Comment by vitabaks** [comment, marked as answer]
Hi @vasilytray !

1. Change IP adress in `cluster_vip` variable (vars/main.yml)
2. Run playbook with tag

for "Type B" (default) scheme

```
ansible-playbook config_pgcluster.yml --tags "vip_manager"
```

Or, for "Type A" scheme
```
ansible-playbook balancers.yml --tags "keepalived"
```

All tags https://github.com/vitabaks/postgresql_cluster/blob/master/tags.md

**Reply by vasilytray** [reply]
thank you very much! This is very cool! and to exchange data in keepalived, I just need to exchange data for downloading ansible-playbook deploy_pgcluster.yml --tags haproxy_conf,confd_conf,pgbouncer_config ?

**Reply by vitabaks** [reply]
Exactly, you are right
for the "Type A"  scheme is a bit more complicated and requires more tags

or just run the entire playbook

```
ansible-playbook balancers.yml
```

**Reply by vitabaks** [reply]
>> pgbouncer_config

if you are using the latest version from the master branch then pgbouncer is already listening to all addresses

`pgbouncer_listen_addr` variable

**Reply by vasilytray** [reply]
thank you very much!
